package.json

```diff
@@ -36,7 +36,7 @@
     "clean": "rm -rf assets/stylesheets build dist node_modules"
   },
   "engines": {
-    "node": "^18",
+    "node": "^v18.15.0",
```

Comment:
omg stop

Reply:
It's a bit overly specific for Node but I don't mind too much (I may even prefer it actually).
Aside from that, the rest looks fine...? Should we approve?

Reply:
i'm against the app enforcing you to have an _exact_ version of node. it's so unnecessary. there's nothing to the app that's sooooo specific to this node version